grpcx/server_test.go

```diff
@@ -55,6 +55,6 @@ func TestWithRequestResponseLogs(t *testing.T) {
 
 	a.NoError(err)
 	a.Equal(expected, resp)
-	a.Contains(w.String(), "request=my-request")
-	a.Contains(w.String(), "response=my-response")
+	a.Contains(w.String(), req)
+	a.Contains(w.String(), expected)
```

Comment:
it would be great if req and expected were objects so that the test would serve as an example of how the log will look like.
you can also take a look at the logx options WithoutTimeOpt and WithoutFileInfo so that you can compare the whole log line to what you expect.